pubspec.yaml

```diff
@@ -25,8 +25,9 @@ dependencies:
 dev_dependencies:
   # These two build deps are required by webdev.
   build_runner: ^2.1.2
-  build_web_compilers: ^3.0.0
-  lints: ^2.0.1
+  build_test: ^2.0.0
```

Comment:
Adding build_test allows gha-dart-oss to actually run tests.

Reply:
I wonder if we should remove the `build_test` check from gha-dart-oss?

if the existence of `build_runner` is the only actual thing that determines whether we should run `dart test` vs `dart run build_runner test`? The additional check seems overkill

Reply:
For this repo, we COULD just run `dart test` but it sees build_runner so tries to choose between pub run builder_runner test or ddev test.
Removing build_test and trying PBRT
```
Bad state: Missing dev dependency on package:build_test, which is required to run tests.

Please update your dev_dependencies section of your pubspec.yaml:

  dev_dependencies:
    build_runner: any
    build_test: any
    # If you need to run web tests, you will also need this dependency.
    build_web_compilers: any
```
and we need build_runner to be able to run webdev. So I'm fine just adding build_runner and leaving that in.